main.nf

```diff
@@ -500,14 +728,15 @@ smn_tsv
 
 // call STRs using ExpansionHunter, and plot alignments with GraphAlignmentViewer
 process expansionhunter {
+	publishDir "/${OUTDIR}/versions", mode: 'copy' , overwrite: 'true', pattern: '*versions.yml'
 	tag "$group"
 	cpus 2
 	time '10h'
 	memory '40 GB'
 	scratch true
 	stageInMode 'copy'
 	stageOutMode 'copy'
-	//publishDir "${OUTDIR}/plots/GAV/${group}", mode: 'copy' , overwrite: 'true', pattern: '*.png'
+	//publishDir "/${OUTDIR}/plots/GAV/${group}", mode: 'copy' , overwrite: 'true', pattern: '*.png'
```

Comment:
Path updated in comment. Should the comment be removed or included?

Reply:
some weird ass code. Don't even know what GAV is. Must have been something Ryan snuck in during testing

Reply:
No I only added `versions` directories. That was there before, I just changed all `"${OUTDIR}` to `/"${OUTDIR}`.